### DISCUSSION TITLE
bibliography missing translator and editor

### DISCUSSION THREAD

**Discussion by YonatanRubin** [typst/typst#4114] [General]
### Description

some properties of a citation are missing from the long form of a bibliography entry while showing in the short form. I found the problem in the translator and editor properties, but it is likely to happen in multiple properties.

this is the bibtex file:
```
@book{test,
  title = {{title}},
  author = {author},
  translator = {translator},
  editor = {editor},
  year = {2024},
  series = {{series}},
  publisher = {publisher},
  address = {city}
}
```

the test file I used:
```typst
#set cite(style: "chicago-fullnotes")
@test
#bibliography("library_test.bib")
```
expected result:
Author. Title. Edited by Editor. Translated by Translator. Series. city: publisher, 2024.

actual result:
![2024-05-10-162638_491x170_scrot](https://github.com/typst/typst/assets/9606379/17e0496a-8a84-4676-a4e2-3e7a816fc9c0)
![2024-05-10-162627_423x200_scrot](https://github.com/typst/typst/assets/9606379/0a5aeb22-9b8e-4394-8827-588018d08d07)

### Operating system

Linux

### Typst version

- [X] I am using the latest version of Typst

**Comment by Enivex** [comment]
When you write `#set cite(style: "chicago-fullnotes")` you're setting only the style for citations, not the bibliography (which here has defaulted to "IEEE"). If you want to set both, use `#set bibliography(style: )` instead
